src/components/Routing.js

```diff
@@ -99,17 +100,27 @@ class Routing extends Component {
         // this.addTranslationsForActiveLanguage(defaultLanguage)
     }
 
-    componentDidMount = () => {
+    componentDidMount = async () => {
         const { refreshAccount, handleRefreshUrl, history, clearAlert, clear, handleRedirectUrl, handleClearUrl } = this.props
         
         handleRefreshUrl()
-        refreshAccount()
-
-        history.listen(() => {
+        try {
+            await refreshAccount()
+        } catch (e) {
+            console.log(e)
+        }
+        
+        history.listen(async () => {
             handleRedirectUrl(this.props.router.location)
             handleClearUrl()
-            refreshAccount()
-
+            try {
+                await refreshAccount()
```

Comment:
why is this change needed?

Reply:
Same: https://github.com/near/near-wallet/pull/700/files/577d401fae66b2ec1ae621feb5b58802a03bd837#r457813385

Reply:
@mattlockyer  hmm, this links to `package.json` changes, not sure what are you referring to

Reply:
sorry, not sure what happened. The code for the history listener has since been updated to reflect that we should not try to refreshAccount if we are in one of the create account flows.